playbooks/roles/l2tp-ipsec/templates/instructions.md.j2

```diff
@@ -44,6 +44,13 @@ You can verify that your traffic is being routed properly by [looking up your IP
 
 <a name="osx"></a>
 ### OS X ###
+
+There are two ways to configure the OS X L2TP VPN to connect to this Streisand server. The easiest is to install an OS X *profile* containing the VPN definition. You can also set it up manually.
+
+The profile is [available here](streisand.mobileconfig).
```

Comment:
We should include additional instructions here. Maybe something like...

"Double-click on the downloaded file to begin the connection setup process."